src/web/app/components/modified-timestamps-modal/modified-timestamps-modal.component.html

```diff
@@ -0,0 +1,52 @@
+<p *ngIf="coursesOfModifiedSession">
```

Comment:
I believe this if statement always return true. Since we default `coursesOfModifiedSession` to an empty array in `modified-timestamps-modal.component.ts` and an empty array is truthy, this if statement is always true. 

I think this is why we have an unintended behavior of this text being displayed for Instructor courses pages, when it's supposed to be the latter one:

<img width="1421" alt="Screen Shot 2023-04-27 at 11 45 12 AM" src="https://user-images.githubusercontent.com/25262042/234755781-329429cf-eed2-413d-9f23-420b1fa7f6d6.png">

A simple fix would be to check for the length instead:

```
<p *ngIf="coursesOfModifiedSession.length !== 0">
```